RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -1787,11 +1789,14 @@ boolean L12_themtharHills()
 			}
 		}
 	}
-
+	
+	// Outside of AG, if we have 3+ effect wishes we'll be wishing for Sinuses for Miles instead
+	boolean considerCloverForInhaler = in_avantGuard() || auto_totalEffectWishesAvailable() < 3;
+	
 	// Target 1000 + 400% = 5000 meat per brigand. Of course we want more, but don\'t bother unless we can get this.
 	float meat_need = 400.00;
 	//count inhaler if we have one or if we have a clover to obtain one and can use one
-	if((item_amount($item[Mick\'s IcyVapoHotness Inhaler]) > 0 || cloversAvailable() > 0) && auto_is_valid($item[Mick\'s IcyVapoHotness Inhaler]))
+	if((item_amount($item[Mick\'s IcyVapoHotness Inhaler]) > 0) || (cloversAvailable() > 0 && considerCloverForInhaler))
```

Comment:
you still need the valid check due to G Lover, unless I'm missing something

Reply:
and I guess you should do the check that the top floor is available too because otherwise you're not going to manage to use a clover below.

Reply:
Done both.